api/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "CommonJS",
+    "lib": ["es2017"],
```

Comment:
Since Nodejs doesn't support `esnext`, we should change this  to
```
    "target": "es6",
    "module": "CommonJS",
```
and remove `lib`.